### PR TITLE
Update .travis.yml to run lex-common and removed ros industrial build

### PR DESCRIPTION
**Pull request by xabxx**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,14 +17,14 @@ env:
...
   - git clone https://github.com/aws-robotics/utils-ros1.git utils-ros1
-  - git clone https://github.com/aws-robotics/cloudwatchlogs-ros1 cloudwatchlogs-ros1
-  - git clone https://github.com/aws-robotics/cloudwatch-common cloudwatch-common
-  - git clone https://github.com/aws-robotics/cloudwatchmetrics-ros1 cloudwatchmetrics-ros1
-  - git clone https://github.com/aws-robotics/health-metrics-collector-ros1 health-metrics-collector-ros1
-  - git clone https://github.com/aws-robotics/monitoringmessages-ros1 monitoringmessages-ros1
-  - git clone https://github.com/aws-robotics/kinesisvideo-ros1 kinesisvideo-ros1
-  - git clone https://github.com/aws-robotics/kinesisvideo-encoder-common kinesisvideo-encoder-common
-  - git clone https://github.com/aws-robotics/kinesisvideo-encoder-ros1 kinesisvideo-encoder-ros1
-  - git clone https://github.com/ros-industrial/industrial_ci.git .ros_ci
```
Comment (AAlon) [inline .travis.yml:28]: You need ci to run ci :-)

**Review by AAlon — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (xabxx) [reply .travis.yml:28]: yep, added that. I thought we wanted to use our build script but just remembered that your repo doesnt have the same ci env. and thus cannot use our build script.

Comment (AAlon) [reply .travis.yml:28]: Our build scripts don't do prerelease testing, that's why we're using industrial-ci.

Comment (xabxx) [reply .travis.yml:28]: yep.

**Pull request merged**